src/main/java/org/reaktivity/nukleus/oauth/internal/stream/OAuthProxyFactory.java

```diff
@@ -315,6 +448,9 @@ private OAuthProxy(
 
                 this.expiryFuture = executor.schedule(delay, MILLISECONDS, targetRouteId, targetStreamId, TOKEN_EXPIRED_SIGNAL);
             }
+
+            this.grant = Objects.requireNonNull(grant);
+            grant.acquire();
```

Comment:
Let's move this into `newInitialStream` as it has a side-effect.
Note: we'll need to call `grant.acquire()` once for initial, and once for reply as part of setting up reply throttle in advance of `newReplyStream`.